src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -151,24 +148,16 @@ public function isCurrent(ItemInterface $item)
      * Checks whether an item is the ancestor of a current item.
      *
      * @param ItemInterface $item
-     * @param integer       $depth The max depth to look for the item
+     * @param int|null      $depth The max depth to look for the item
      *
-     * @return boolean
+     * @return bool
      */
-    public function isAncestor(ItemInterface $item, $depth = null)
+    public function isAncestor(ItemInterface $item, ?int $depth = null): bool
     {
         if (null === $this->matcher) {
             throw new \BadMethodCallException('The matcher must be set to get the breadcrumbs array');
         }
 
         return $this->matcher->isAncestor($item, $depth);
     }
-
-    /**
-     * @return string
-     */
-    public function getName()
```

Comment:
We need to keep this method as long as we support Twig 1.

Reply:
It's deprecated since Twig 1.26, so we can safely remove it

Reply:
Right, `AbstractExtension` has a default implementation for that method. I didn't know that. Good catch. :-)